src/components/dashboard/FeedModalList.js

```diff
@@ -72,49 +75,67 @@ const FeedModalList = ({
       // https://stackoverflow.com/questions/40200660/react-native-scrollto-with-interactionmanager-not-working
       setTimeout(() => {
         flatListRef && flatListRef.current && flatListRef.current.scrollToOffset({ animated: false, offset })
+        setLoading(false)
       }, 0)
     }
-  }, [offset, flatListRef])
-
-  const getItemLayout = (_: any, index: number) => {
-    const length = screenWidth
-    return { index, length, offset: length * index }
-  }
-
-  const renderItemComponent = ({ item, separators, index }: ItemComponentProps) => (
-    <View style={styles.horizontalListItem}>
-      <FeedModalItem
-        navigation={navigation}
-        item={item}
-        separators={separators}
-        fixedHeight
-        onPress={() => handleFeedSelection(item, false)}
-      />
-    </View>
+  }, [offset, flatListRef, setLoading])
+
+  const getItemLayout = useCallback(
```

Comment:
isn't screenWidth a fixed value ? event if running on desktop the app has mobile-like viewport and resizing browser window don't have such influence

so probably

1. determine screenWidth outside component
2. remove it from the deps everywhere in this cokmponent
3. move getItemLayout outside component